sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/EditContactPage.java

```diff
@@ -44,4 +49,50 @@
   public static final By CONTACT_CREATED_POPUP = By.cssSelector(".v-Notification-caption");
   public static final By DELETE_BUTTON = By.id("delete");
   public static final By DELETE_POPUP_YES_BUTTON = By.cssSelector(".popupContent #actionConfirm");
+  public static final By CONTACT_CLASSIFICATION = By.cssSelector("#contactClassification label");
+  public static final By MULTI_DAY_CONTACT_CHECKBOX = By.cssSelector("#multiDayContact > input");
+  public static final By MULTI_DAY_CONTACT_LABEL = By.cssSelector("#multiDayContact > label");
+  public static final By FIRST_DAY_CONTACT_DATE = By.cssSelector("#firstContactDate input");
+  public static final By EXTERNAL_ID_INPUT = By.id("externalID");
+  public static final By EXTERNAL_TOKEN_INPUT = By.id("externalToken");
+  public static final By REPORTING_DISTRICT_COMBOBOX =
+      By.cssSelector(".v-verticallayout [location='reportingDistrict'] [role='combobox'] div");
+  public static final By IDENTIFICATION_SOURCE_INPUT =
+      By.cssSelector("#contactIdentificationSourceDetails");
+  public static final By CONTACT_IDENTIFICATION_SOURCE_DETAILS_COMBOBOX =
+      By.cssSelector(
+          ".v-verticallayout [location='contactIdentificationSource'] [role='combobox'] div");
+  public static final By PROHIBITION_TO_WORK_OPTIONS = By.cssSelector("#prohibitionToWork label");
+  public static final By HOME_BASED_QUARANTINE_OPTIONS =
+      By.cssSelector("#quarantineHomePossible label");
+  public static final By QUARANTINE_COMBOBOX =
+      By.cssSelector(".v-verticallayout [location='quarantine'] [role='combobox'] div");
```

Comment:
Please replace with #quarantine div

Reply:
done. tried to rework other comboboxes but then the test was failing